rdtools/test/plotting_test.py

```diff
@@ -162,3 +165,51 @@ def test_soiling_rate_histogram_kwargs(soiling_info):
     )
     result = soiling_rate_histogram(soiling_info, **kwargs)
     assert_isinstance(result, plt.Figure)
+
+
+@pytest.fixture()
+def clipping_power_degradation_signal():
+    clipping_power_series = pd.Series(np.arange(1, 101))
+    #Add datetime index to second series
+    time_range = pd.date_range('2016-12-02T11:00:00.000Z', '2017-06-06T07:00:00.000Z', freq='H')
+    clipping_power_series.index = pd.to_datetime(time_range[:100])
+    return clipping_power_series
+
+
+@pytest.fixture()
+def clipping_info(degradation_power_signal):
+    '''
+    Return results of clipping filter applied to a degradation signal.
+
+    Returns
+    -------
+    signal_filtered: Pandas series, filtered degradation power signal
+    clipping_mask_series: Pandas series, boolean mask time series for 
+    clipping, with True indicating a clipping period and False 
+    representing a non-clipping period
+    '''
+    signal_filtered, clipping_mask_series = geometric_clip_filter(clipping_power_degradation_signal)
+    return signal_filtered, clipping_mask_series
+
+
+@pytest.fixture()
+def test_clipping_filter_plots(clipping_info):
+    signal_filtered, clipping_mask_series = clipping_info
+    
+    # test defaults
+    result = tune_clip_filter_plot(clipping_power_degradation_signal,
+                                    clipping_mask_series)
+    assert_isinstance(result, plotly.graph_objs._figure.Figure)
+
+
+def test_clipping_filter_plots_kwargs(clipping_info):
+    signal_filtered, clipping_mask_series = clipping_info
+
+    # test kwargs
+    kwargs = dict(
+        display_web_browser = True
+    )
+    result = tune_clip_filter_plot(clipping_power_degradation_signal,
```

Comment:
Probably need to add `clipping_power_degradation_signal` as a parameter of this test function too

Reply:
Ah, will give that a shot. Thanks for the tip @kanderso-nrel I've been trying to debug this for the past twenty minutes trying to figure out why the Travis CI build was failing